WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -126,7 +131,7 @@ public final class WordPressComOAuthClient: NSObject {
     @objc public func authenticateWithUsername(_ username: String,
                                                password: String,
                                                multifactorCode: String?,
-                                               needsMultifactor: @escaping (_ userID: Int, _ nonceInfo: SocialLogin2FANonceInfo) -> Void,
+                                               needsMultifactor: ((_ userID: Int, _ nonceInfo: SocialLogin2FANonceInfo) -> Void)? = .none,
```

Comment:
Why not providing a noop closure instead?
```suggestion
                                               needsMultifactor: @escaping (_ userID: Int, _ nonceInfo: SocialLogin2FANonceInfo) -> Void = { _ in },
```

Or was this explicitly because you wanted to handle the case of calling `failure` with an error if no closure was provided explicitly?

Reply:
> Or was this explicitly because you wanted to handle the case of calling failure with an error if no closure was provided explicitly?

That was the intention. In order to communicate to out-of-date clients that they should do _something_, but without a breaking change or crashing at runtime.